shpc/main/settings.py

```diff
@@ -114,7 +114,12 @@ def get(self, key, default=None):
         """
         Get a settings value, doing appropriate substitution and expansion.
         """
-        value = self._settings.get(key, default)
+        # This is a reference to a dictionary (object) setting
+        if ":" in key:
+            key, subkey = key.split(":")
```

Comment:
```suggestion
            subkey, value = value.split(":")
```

Reply:
actually let me just edit these locally and add a test - need a few minutes getting ready for bed! I think we might not need to do this, we don't care about the value for get.